.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea
-node_modules
\ No newline at end of file
+node_modules
```

Comment:
I don’t know if chip would like this. He loves to commit node modules

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA